### PR TITLE
Add instructions for ef migrations to readme.

### LINKED ISSUE #764: Identity Scaffolding: Update readme about ef migrations
When the user creates a new DbContext, the readme should include information about running migrations, specifically:
 - dotnet ef migrations add initial
 - dotnet ef database update

### PR DESCRIPTION
**Pull request by prafullbhosale**
Fixes #764 

cc @vijayrkn @mkArtakMSFT @javiercn

### CODE REVIEW

**Comment by javiercn** [comment]
@ajcvickers Would it be OK to have both instructions? This is usable from command line, so we want to have instructions that work cross-platform
* Powershell
* dotnet ef

If you guys can provide the specific instructions, @seancpeters will be happy to update the PR with your recommendation

**Comment by ajcvickers** [comment]
The generated database code requires Entity Framework Core Migrations. Run the following commands:
+@:1. dotnet ef migrations add CreateIdentitySchema
+@:2. dotnet ef database update

Or from the Visual Studio Package Manager Console:
+@:1. Add-Migration CreateIdentitySchema
+@:2. Update-Database

File: src/VS.Web.CG.Mvc/Templates/Identity/ScaffoldingReadme.cshtml [hunk 1]
```diff
@@ -21,7 +21,15 @@ To use ASP.NET Core Identity you also need to enable authentication. To authenti
 The generated UI requires MVC. To add MVC to your app:
 1. Call services.AddMvc() from your ConfigureServices method
 2. Call app.UseMvc() from your Configure method (after authentication)
-
+@{
+    if (!Model.IsUsingExistingDbContext)
+    {
+@:
+@:The generated database code requires Entity FrameworkCore Migrations. Run the following commands:
```
Comment (divega) [inline src/VS.Web.CG.Mvc/Templates/Identity/ScaffoldingReadme.cshtml]: Space missing between Framework and Core.

**Comment by prafullbhosale** [comment]
Updated the content.

**Comment by prafullbhosale** [comment]
>but that the commands should include -o <<PathToDbContext>>\Migrations to match the convention followed on the templates.

Have the templates changed recently? With SDK version `2.1.300-preview3-008646` I still see the migrations for Individual Auth template are under `<ProjectRoot>\Data\Migrations`

**Pull request merged**